bookdown/preamble.tex

```diff
@@ -1,17 +1,23 @@
 \usepackage{booktabs}
 \usepackage{longtable}
-\usepackage[bf,singlelinecheck=off]{caption}
-% \usepackage[paperwidth=6in, paperheight=9in]{geometry}
 
-% \setmainfont[UprightFeatures={SmallCapsFont=AlegreyaSC-Regular}]{Alegreya}
+\usepackage[margin=1in]{geometry}
+\usepackage{fontspec}
+\setmonofont{Fira Code}[
```

Comment:
Why can't we stick to a standard font?

Reply:
We now use the default serif font and `Dejavusans mono` for monospace.